spec/unit/warehouse_spec.rb

```diff
@@ -4,20 +4,20 @@
   let(:empty_warehouse) { Warehouse.new(product_id: nil, amount: nil) }
   let(:warehouse) { Warehouse.new(product_id: 1, amount: 10) }
 
-  it "has 2 params" do
+  it "Warehouse should have 2 params" do
     expect{
       empty_warehouse
     }.to_not raise_error(ArgumentError)
   end
 
-  context "#product_id " do
-    it "returns 1 when 1 is given" do
+  describe "#product_id " do
+    it "should returns 1 when 1 is given" do
       expect(warehouse.product_id).to eql(1)
     end
   end
 
-  context "#amount " do
-    it "returns 10 when 10 is given" do
+  describe "#amount " do
+    it "should returns 10 when 10 is given" do
```

Comment:
do not use should word on the beginning of test description. You should not ask, or please test to work ;]